cwl/tools/copy_number_consensus_call.cwl

```diff
@@ -0,0 +1,26 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: copy_number_consensus_call
+requirements:
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: 1000
+  - class: DockerRequirement
+    dockerPull: 'zhangb1/data-assembly'
```

Comment:
same concern that Miguel had for the other dockers being pulled from DockerHub

Reply:
This would be  temporary in my docker hub, I will move that after the workflow fully wrap

Reply:
Ah, @zhangb1 , that will happen in another PR then?